lib/fs.js

```diff
@@ -2,7 +2,10 @@
 
 const fs = require('fs');
 const path = require('path');
-const MemoryFileSystem = require('memory-fs');
+const { fs: memfs } = require('memfs');
+// borrow join and normalize from memory-fs
+memfs.join = require('memory-fs/lib/join');
+memfs.normalize = require('memory-fs/lib/normalize');
```

Comment:
It is not good solution, we don't need two memory-fs package

Reply:
Those functions appear to be unrelated to the file system itself and could live somewhere else.

Reply:
I know, but it is bad have two memory fs package, we should move these function to separate repo or move to `memfs`